lib/rubocop/cop/ezcater/rspec_require_browser_mock.rb

```diff
@@ -0,0 +1,56 @@
+module RuboCop
+  module Cop
+    module Ezcater
+      # Enforce use of `mock_ezcater_app`, `mock_chrome_browser` & `mock_custom_browser` helpers
+      # instead of mocking `Browser` or `EzBrowser` directly.
+      #
+      # @example
+      #
+      #   # good
+      #   mock_ezcater_app
+      #   mock_chrome_browser
+      #   mock_custom_browser
+      #   mock_ezcater_app(device: "My Device", version: "1.2.3", language: "en=US,en")
+      #   mock_chrome_browser(device: "My Device", version: "1.2.3", language: "en=US,en")
+      #   mock_custom_browser(user_agent: "My Custom Agent", language: "en=US,en"
+      #
+      #   # bad
+      #   allow(Browser).to receive...
+      #   allow(EzBrowser).to receive...
+      class RspecRequireBrowserMock < Cop
+        EZ_BROWSER_MSG = "Use the mocks provided by `BrowserHelpers` instead of mocking `allow(EzBrowser)`".freeze
```

Comment:
Suggestion: "instead of mocking `EzBrowser` directly". You're not mocking the `allow`.